api/v1beta1/groupversion_info.go

```diff
@@ -25,15 +25,15 @@ import (
 )
 
 const (
-	GroupName     = "lm-eval-service.github.com"
+	GroupName     = "foundation-model-stack.github.com.github.com"
 	Version       = "v1beta1"
-	KindName      = "EvalJob"
-	FinalizerName = "lm-eval-service.github.com/finalizer"
+	KindName      = "LMEvalJob"
+	FinalizerName = "lm-eval-service.foundation-model-stack.github.com.github.com/finalizer"
```

Comment:
NIT: Since you're on this line, should the finalizer be `lm-eval-job...` instead of `lm-eval-service...`? I doubt it makes much difference, but it would seem more consistent.

Reply:
fixed